Makefile

```diff
@@ -56,6 +56,9 @@ bindata_assetfs.go: .build/deps.ok .build/bindata_assetfs.$(GO_BINDATA_MODE) $(A
 $(NAME): .build/deps.ok .build/vendor.ok bindata_assetfs.go $(SOURCES)
 	go build -ldflags "-X main.version=$(VERSION)"
 
+$(NAME)-static: .build/deps.ok .build/vendor.ok bindata_assetfs.go $(SOURCES)
```

Comment:
We're no longer keeping resulting binary in the image it was build, so we need a more static build.
But it's likely that `CGO_ENABLED=0` is only needed really, @terinjokes ?